examples/mem-alloc-test/src/main.rs

```diff
@@ -166,22 +164,30 @@ fn worker_thread(
         let start_time = Instant::now();
 
         /* Create an array of x GB where x is a random number between 1 to 4 */
-        let mut large_vector = Vec::with_capacity(size);
-        large_vector.resize(size, 0);
+
+        // Create a layout based on the size and alignment
+        let align = get_random_num(2, PAGE_SIZE).next_power_of_two();
```

Comment:
This works, but has a slight bias toward larger alignments. If you want the different alignments to have the same probability, you could do something like generating a number X between 0 and 12 inclusive, and then using the alignment (1 << X). Not a big deal.